### PR TITLE
Logging for server and OneSDK resolver

### PR DESCRIPTION
**Pull request by freaz**
Adds pino to write structured logs for server and onesdk performs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -216,6 +216,44 @@ OneService can be deployed to [Heroku with Git](https://devcenter.heroku.com/art
...
+## Logs
+
+OneService can print structured logs. To allow it use `--logs=<level>` flag:
+
+```shell
+oneservice --logs
+oneservice --logs=trace
+```
+
+Set optional level specifier to the desired logging level. In order of priority, available levels are:
```
Comment (zdne) [inline README.md:228]: Nit

```suggestion
Set the optional logging level specifier. Available levels are (ordered by priority) :
```

File: README.md [hunk 2]
```diff
@@ -216,6 +216,44 @@ OneService can be deployed to [Heroku with Git](https://devcenter.heroku.com/art
...
+
+- 'fatal'
+- 'error'
+- 'warn'
+- 'info'
+- 'debug'
+- 'trace'
+
+The logging level is a __minimum__ level. For instance if `logger.level` is `'info'` then all `'fatal'`, `'error'`, `'warn'`, and `'info'` logs will be enabled.
```
Comment (zdne) [inline README.md:237]: Nit

```suggestion
The logging level is the __minimum__ threshold. When set, every higher level will be logged as well. For instance if `logger.level` is `'info'` then all `'fatal'`, `'error'`, `'warn'`, and `'info'` logs will be enabled.
```

File: src/cli.ts [hunk 3]
```diff
@@ -1,23 +1,31 @@
...
-import { Command } from 'commander';
+import { Command, InvalidOptionArgumentError } from 'commander';
 import { bootstrap } from '.';
+import { Level } from './logger';
 
 const program = new Command();
 program.version(packageJson.version);
 
+function validateLogLevel(level: string): Level | undefined {
+  if (!['fatal', 'error', 'warn', 'info', 'debug', 'trace'].includes(level)) {
```
Comment (zdne) [inline src/cli.ts:12]: missing `silent` ?

**Review by zdne — COMMENTED** [review]
Sanity checked, perhaps missing `silent` in args validation?

File: README.md (hunk 1 above)
Comment (freaz) [reply README.md]: Fixed

File: src/cli.ts (hunk 3 above)
Comment (freaz) [reply src/cli.ts]: Damn you are right. Fixed.

File: README.md (hunk 2 above)
Comment (freaz) [reply README.md]: Fixed

**Review by zdne — APPROVED** [review]
LGTM

**Pull request merged**